pontoon/base/static/css/translate.css

```diff
@@ -1087,6 +1087,11 @@ body > header aside p {
   padding-left: 2%;
 }
 
+#entitylist > .wrapper > ul > li p span.translation-string:lang(mk),
+#entitylist > .wrapper > ul > li p span.translation-string:lang(sr) {
```

Comment:
Please use the standard [attribute selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors):
`#entitylist > .wrapper > ul > li p span.translation-string[lang="sr"]`

Surprisingly to me, your _colon_ syntax also works fine (at least in Firefox), but let's stick to the standard.